highfidelity/hifi
# FogBugz 7425 - Billboard orientation and grabber move up tool are now dependent on the Avatar referential.

FogBugz
7425 IPD Secondary Task List: edit.js arrow orientation: 
    - Grabber Move Up Tool and gizmo arrow now work as expected, taking into account the orientation of the Avatar.
Update CREATE affordances to be correctly oriented on wiki-planet
    - Billboards now take into account the orientation of the Avatar.

---------------------------------------------------------------------------

Test plan:
In HMD mode:
1. Go to a domain where you have edit power.
2. Open the "Create" mode and spawn a cube.
3. Notice the presence of the "grabber move up" gizmo.
![grabbermoveup](https://user-images.githubusercontent.com/403611/31948457-ead3fcc4-b8ce-11e7-92d9-9f114c62ea32.PNG)

Try to look at it from different angles. 

Expected result: The gizmo should always be oriented towards you.

4. Try to use the tool to move the cube up and down.

Expected result: You are able to move the cube up and down.

-----------------------------------------------------------------------
5. Go to wikiplanet/newyork (you might need edit permission)
6. Open the "Create" mode and spawn a cube.
7. Notice the presence of the "grabber move up" gizmo. Try to look at it from different angles. 
Expected result: The gizmo should always be oriented towards you.
8. Try to use the tool to move the cube up and down.
Expected result: You are able to move the cube up and down.
-----------------------------------------------------------------------
In Desktop mode, repeat the steps for the HMD test.
